File: beginner/javascript/1015.js
var input = require('fs').readFileSync('stdin', 'utf8');

var linhas = input.split('\n');

var linha1 = linhas[0].split(' ');
var linha2 = linhas[1].split(' ');

var x1 = parseFloat(linha1[0]);
var y1 = parseFloat(linha1[1]);
var x2 = parseFloat(linha2[0]);
var y2 = parseFloat(linha2[1]);

var distancia = Math.sqrt(Math.pow(x2 - x1, 2) + Math.pow(y2 - y1, 2));

console.log(distancia.toFixed(4));

File: beginner/java/1015.java
import java.io.IOException;
import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);

            scanner.useLocale(Locale.US);

            double x1 = scanner.nextDouble();
            double y1 = scanner.nextDouble();
            double x2 = scanner.nextDouble();
            double y2 = scanner.nextDouble();

            double distancia = Math.sqrt(Math.pow(x2 - x1, 2) + Math.pow(y2 - y1, 2));

            System.out.printf("%.4f%n", distancia);

        scanner.close();
    }
}
